src/scss/_tabs.scss

```diff
@@ -5,10 +5,13 @@ $jazz-tab-border-size: 4;
   display: flex;
   border: none;
   // bottom border serves as a rail across the entire set of tabs
-  border-bottom: px-to-rem($jazz-tab-border-size) solid jazz-theme(tablist-border-color);
+  border-bottom: px-to-rem($jazz-tab-border-size) solid
+    jazz-theme(tablist-border-color);
 }
 
-.jazz-tablist[role="tablist"] button[role='tab'] {
+.jazz-tablist[role="tablist"] button[role="tab"],
+.jazz-tablist[role="tablist"] a[role="tab"],
+.jazz-tablist[role="tablist"] span {
```

Comment:
Should the span selector be scoped to [role="tab"] also?  If so, I wonder if the selector can be made more generic (*[role="tab"])?

Reply:
Yes, the span selector can and probably should have the scoping `[role="tab"]` added.

And I also think a more generic selector `*[role="tab"]` would work, assuming that no other components, tags or elements in the page will also have the `role="tab"` property.

Reply:
Keep in mind that the scope here is already within the ".jazz-tablist" class, so you don't have to worry about the rest of the page.  I am not sure if the "*" syntax that I provided is correct, so you will have to try it to make sure that it works as we want it to here.  Thanks!